.lintr

```diff
@@ -6,6 +6,8 @@ linters: linters_with_tags(
     function_argument_linter = NULL,
     indentation_linter = NULL,
     object_name_linter = NULL,
+    object_usage_linter = NULL,
```

Comment:
I feel like it would be better not to do these things vs turning them off? I would at least open an issue for tracking

Reply:
yeah, i need to circle back and address the linting anyway